app/js/controllers/foyer/individuForm.js

```diff
@@ -37,38 +36,35 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         }
     ];
 
-    $scope.individu = {
-        nationalite: 'fr',
+    var defaultIndividu = {
+        nationalite: 'ue',
         assPreconditionRemplie: false,
         scolarite: 'college',
         tauxInvalidite: 'moins50',
         boursier: false,
         aCharge: true,
         place: false,
-        role: individuRole
+        titreSejour10ans: true,
+        titreSejour5ans: true,
+        role: individuRole,
+        situationsPro: []
```

Comment:
Ok, got it, you iterate down there. Good implementation, makes the expected data structure much more readable!  :)